cassandra-protocol/src/types/floats.rs

```diff
@@ -0,0 +1,5 @@
+use ordered_float::OrderedFloat;
+
+pub type Double = OrderedFloat<f64>;
```

Comment:
Sorry, but this is a **very** breaking change. We should not enforce using such wrappers in library code.

Reply:
Floats can't be hashed though? So we might as well go back to representing the sets as vecs instead.

Reply:
Not directly, no. While it might be useful to have hashing ability at this point, we cannot enforce such thing on other users. Removing primitive floats from here would break literally every code that uses them.